### DISCUSSION TITLE
这个项目的定位是什么？

### DISCUSSION THREAD

**Discussion by sre-show** [opsre/go-ldap-admin#41] [Q&A] [answered]
如题

**Comment by eryajf** [comment, marked as answer]
参考官方文档：[这个项目是什么](http://ldapdoc.eryajf.net/pages/101948/#%E8%BF%99%E4%B8%AA%E9%A1%B9%E7%9B%AE%E6%98%AF%E4%BB%80%E4%B9%88)

go-ldap-admin旨在为openLDAP服务端提供一个简单易用，清晰美观的现代化管理后台。

在完成针对openLDAP的管理能力之下，我们希望平台能够成为企业IM与应用之间的桥梁，亦即，对钉钉，企业微信，飞书的集成，用户可以选择手动或者自动同步组织架构以及员工信息到平台中，让go-ldap-admin项目成为打通企业IM与企业内网应用之间的桥梁。

我给项目绘制的架构简图如下：

![](https://camo.githubusercontent.com/65fe95b5ee953a83c2bbd241ca8ef897805ea34f6b98571c04d4f4ceabdd7bed/68747470733a2f2f63646e2e737461746963616c792e636f6d2f67682f657279616a662f74752f6d61696e2f696d672f696d6167655f32303232303631345f3133313532312e6a7067)
